cmd/thanos/query.go

```diff
@@ -663,30 +651,39 @@ func runQuery(
 		EnableAtModifier:     true,
 	}
 
+	thanosEngineOpts := promql.EngineOpts{
```

Comment:
Do we need to define this separately? I think we'd just be able to use engineOpts directly! 🙂